src/main/java/com/example/translate/controller/TranslationsController.java

```diff
@@ -94,7 +94,7 @@ public ResponseEntity<Translations> updateData(@PathVariable Long id,
         return ResponseEntity.ok(updatedData);
     }
 
-    @PutMapping(value = "/new", consumes = {MediaType.APPLICATION_JSON_VALUE})
+    @PostMapping(value = "/new", consumes = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
when you give path value to your api no need first slash "/" you can basically do value="new"